snmp/datadog_checks/snmp/data/default_profiles/_dell-generic-if.yaml

```diff
@@ -0,0 +1,3 @@
+extends:
+  - _dell.yaml
+  - _generic-if.yaml
```

Comment:
I think we should avoid having too many profiles.

It might be better to not have:
-_dell-generic-if.yaml
- _dell-intel-lan-adapters.yaml

And have the leaf profiles directly import:
- _dell.yaml
- _generic-if.yaml
- _intel-lan-adapters.yaml